packages/cli/utils/json.go

```diff
@@ -59,27 +61,15 @@ func ApplyFunctionToJSON(messageBody string, modifier func(interface{}) interfac
 	}
 }
 
-func InferTemplateEnvVarFromStatus(jobStatus string) (string, error) {
```

Comment:
@EricRibeiro I wanted to get you to look over this change - it looked like we were previously relying on environment variables pre-loaded with the default template contents, but that wasn't working with the current golang implementation. Can you read over the changes in this file and let me know if this causes issues with the way it's supposed to work?